Failure of macOS (EXPERIMENTAL) (3.8) checks on main

### Description

Likely within the past week, a merged PR is causing failures in main w.r.t. the `macOS (EXPERIMENTAL) (3.8)` check. Until corrected, all new PR's and existing PR's merged with main are affected. The following failures are occuring:

```
FAILED watertap/property_models/tests/test_cryst_prop_pack.py::TestNaClPropertySolution_4::test_property_regression - ValueError: Cannot load a SolverResults object with bad status: error
FAILED watertap/unit_models/tests/test_electroNP_ZO.py::TestElectroNP::test_solution - assert 79571089844804.64 == 452890000000000.0 ± 4.5e+10
  comparison failed
  Obtained: 79571089844804.64
  Expected: 452890000000000.0 ± 4.5e+10
FAILED watertap/unit_models/tests/test_electroNP_ZO.py::TestElectroNP::test_costing - ValueError: Cannot load a SolverResults object with bad status: error
```

--- a/.github/workflows/checks.yml
+++ b/.github/workflows/checks.yml
@@ -292,7 +292,7 @@ jobs:
         echo '::endgroup::'
     - name: Install Ipopt from conda-forge
       run:
-        conda install --quiet --yes -c conda-forge ipopt
+        conda install --quiet --yes -c conda-forge ipopt=3.14.11
     - name: Build Pyomo extensions
       run: |
         conda install --quiet --yes cmake

